### DISCUSSION TITLE
Why is 1.15 taking so long?

### DISCUSSION THREAD

**Discussion by ItsEnderium** [ScratchAddons/ScratchAddons#2536] [Help] [answered]
1.14.3 is the most recent version of Scratch Addons. I'm wondering why it's taking so long.
There hasn't been a single version of Scratch Addons that I know of that has more patches than 1,14. Is there an addon in the works, such as font customisation or website dark theme with customisable colours?

**Comment by cobaltt7** [comment]
> There hasn't been a single version of Scratch Addons that I know of that has more patches than 1,14

Correct.

1.14.1 was because recommended addons weren't working.
1.14.2 was because you could not access your own unshared projects anymore.
1.14.3 was because we _thought_ the Scratch Team was releasing official 3.0-designed studios (they didn't) and we didn't want addons to break.

So now we want to wait a bit before releasing another version since we just had so many in a row. We normally wait two weeks between releases, so it'd probably be another week/week and a half before 1.15.0.

**Comment by WorldLanguages** [comment, marked as answer]
The v1.15.0 milestone has been set to May 21, 2021 on May 9th (10 days ago). Just another 3 week release instead of 2, not strange at all
